.gitignore

```diff
@@ -1,4 +1,4 @@
-client
+client/activity.*
```

Comment:
Are we sure this fixes ignoring the source, or does it need a leading / in front?

Reply:
> If there is a separator at the beginning or middle (or both) of the pattern, then the pattern is relative to the directory level of the particular .gitignore file itself. Otherwise the pattern may also match at any level below the .gitignore level.
-- [gitignore documentation](https://git-scm.com/docs/gitignore)